src/style/themes/classic/classic-theme.config.js

```diff
@@ -1,30 +1,54 @@
 import { THEMES } from '..';
 
 export default (palette) => {
-  return (
-    {
-      name: THEMES.classic,
+  return {
+    name: THEMES.classic,
+    colors: {
+      base: palette.productGreen,
+      primary: palette.productGreenShade(21),
+      secondary: palette.productGreenShade(41),
+      tertiary: palette.productGreenShade(61),
 
-      colors: {
-        base: palette.productGreen,
-        primary: palette.productGreenShade(21),
-        secondary: palette.productGreenShade(41),
-        tertiary: palette.productGreenShade(61),
+      // generic
+      baseBlue: '#255BC7',
+      greyDarkBlue50: '#8099a4'
+    },
 
-        // generic
-        baseBlue: '#255BC7',
-        greyDarkBlue50: '#8099a4'
-      },
+    status: {
```

Comment:
Where have all of these colours come from? I feel like you should try bringing the classic theming config closer to the component.